### PR TITLE
test(e2e): meeting control bar is not visible after meeting is left

### PR DESCRIPTION
**Pull request by BenjaminGolya**

### CODE REVIEW

File: tests/WebexMeeting.e2e.js [hunk 1]
```diff
@@ -87,9 +85,17 @@ describe('Meeting Widget', () => {
     expect(MeetingPage.muteVideoBtn).toBeVisible();
   });
 
+  it('meeting control bar is visible before meeting has been left', () => {
```
Comment (lalli-flores) [inline tests/WebexMeeting.e2e.js:88]:
```suggestion
  it('meeting control bar is visible before meeting is left', () => {
```

File: tests/WebexMeeting.e2e.js [hunk 2]
```diff
@@ -87,9 +85,17 @@ describe('Meeting Widget', () => {
...
+    expect(MeetingPage.controlBar).toBeVisible();
+  });
+
   it('leaves the meeting', () => {
     MeetingPage.leaveMeetingBtn.click();
     MeetingPage.meetingWidget.waitForDisplayed({timeout: 10000});
     expect(MeetingPage.meetingWidget).toHaveTextContaining('You\'ve successfully left the meeting');
   });
+
+  it('meeting control bar is not visible after meeting has been left', () => {
```
Comment (lalli-flores) [inline tests/WebexMeeting.e2e.js:98]:
```suggestion
  it('meeting control bar is not visible after meeting is left', () => {
```

File: tests/WebexMeeting.e2e.js [hunk 3]
```diff
@@ -15,7 +15,6 @@ describe('Meeting Widget', () => {
         user = browser.call(() => createUser());
         accessToken = user.token.access_token;
       }
-
```
Comment (lalli-flores) [inline tests/WebexMeeting.e2e.js:18]: any reason for removing the new lines here and [here](https://github.com/webex/widgets/pull/96/files#diff-f5d4a1540edc79b8cc150656aaa0a676d13340692408b2aa279a92c96ec973e9L43)?

**Review by lalli-flores — CHANGES_REQUESTED** [review]
Can you update your commit message for better readability?
```
test(e2e): meeting control bar is not visible after meeting is left
```

File: tests/WebexMeeting.e2e.js (hunk 3 above)
Comment (BenjaminGolya) [reply tests/WebexMeeting.e2e.js:18]: I removed the spaces on line 28
I thought there was no need that empty space

**Review by cipak — APPROVED** [review]

File: tests/WebexMeeting.e2e.js [hunk 4]
```diff
@@ -92,4 +96,8 @@ describe('Meeting Widget', () => {
     MeetingPage.meetingWidget.waitForDisplayed({timeout: 10000});
     expect(MeetingPage.meetingWidget).toHaveTextContaining('You\'ve successfully left the meeting');
   });
+
+  it('doesn\'t display any control after leaving', () => {
```
Comment (cipak) [inline tests/WebexMeeting.e2e.js:100]: after leaving the meeting

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**